bower.json

```diff
@@ -18,8 +18,10 @@
     "purescript-console": "^2.0.0",
```

Comment:
Sure, the point is it is only used in the unit tests, therefore it should be devDependency. Basically the dependencies section should be what is required to compile `src/` and not `test/`.